### PR TITLE
Add whole project

### PR DESCRIPTION
**Pull request by bekjon-me**
- added html codes

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    Hello Camper! Hello asd
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```
Comment (bekjon-me) [inline index.html:86]: kindly remove these commented lines

File: index.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <main>
```
Comment (bekjon-me) [inline index.html:10]: it would be great if you use a section tag instead of main

**Review by bekjon-me — COMMENTED** [review]
Good job so far. but you have to fix some codes.

File: index.html (hunk 2 above)
Comment (Qoosim) [reply index.html:10]: - [OPTIONAL] Good job in performing code review on your previous project. You've done pretty well. However, to make your review look more professional, you should always state the reasons for asking the students to make a change on their code. This is useful for more clarification and a quick way for the students to learn some tips which they may not be opportune to come across easily. 
- [OPTIONAL] It is also a best practice to use a couple of emojis and/or gifs in the review. They make the code review more human-friendly. Good job once again for performing a wonderful code review. :+1:

File: index.html (hunk 1 above)
Comment (Qoosim) [reply index.html:86]: - [OPTIONAL] Good job in performing code review on your previous project. You've done pretty well. However, to make your review look more professional, you should always state the reasons for asking the students to make a change on their code. This is useful for more clarification and a quick way for the students to learn some tips which they may not be opportune to come across easily. 
- [OPTIONAL] It is also a best practice to use a couple of emojis and/or gifs in the review. They make the code review more human-friendly. Good job once again for performing a wonderful code review. :+1:

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @bekjon-me, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

_**Kindly check some optional comments I left under the review**_ :pray: 

## Highlights

-  Code Reviewe Performed. :heavy_check_mark: 
-  Correct use of inline comments. :heavy_check_mark: 
-   Good use of polite comments. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**